docs/content.zh/docs/get-started/introduction.md

```diff
@@ -71,27 +66,25 @@ pipeline:
   parallelism: 2
 ```
 
-By submitting the YAML file with `flink-cdc.sh`, a Flink job will be compiled
-and deployed to a designated Flink cluster. Please refer to [Core Concept]({{<
-ref "docs/core-concept/data-pipeline" >}}) to get full documentation of all
-supported functionalities of a pipeline.
+通过使用 `flink-cdc.sh` 提交 YAML 文件，一个 Flink 作业将会被编译并部署到指定的 Flink 集群。
+By submitting the YAML file with `flink-cdc.sh`, 请参考 [核心概念]({{<
+ref "docs/core-concept/data-pipeline" >}}) 以获取 Pipeline 支持的所有功能的完整文档说明。
```

Comment:
Here are some errors with some not translated words.

Reply:
Thanks, I forget to delete this. Done it now.